kitsune/wiki/views.py

```diff
@@ -49,6 +49,7 @@
     send_reviewed_notification, schedule_rebuild_kb,
     send_contributor_notification, render_document_cascade)
 
+from kitsune.lib.sumo_locales import LOCALES
```

Comment:
A small nit: We organize imports into several groups, each sorted. This would go in the `kitsune` group right above this, and should be sorted. I think that means it would be at about line 44.